### ISSUE TITLE
Explore reported management from e-loket

### ISSUE THREAD

**Issue by SanderDevisscher** [inbo/aspbo#281]
Since we get data from hunters concerning management of IAS via the WIZ dataflow (https://github.com/inbo/backoffice-wild-analyse/issues/1128) we need to look into what we want log for internal use as well as what we want to display on the exotenportaal. 

The data  from the old e-loket is currently logged in this file https://drive.google.com/file/d/1r0jQbNqMPmbb6ZFrMAxf-Y7A8_VfF9Mk/view?usp=sharing on the google drive. 
While the data from the WIZ-app is currently logged in this file: https://drive.google.com/file/d/1G2bgrhlphVw7Nfg0_b3P9BBh5q0q1BMO/view?usp=sharing

**Comment by b-dhondt** [comment]
@SanderDevisscher : I see that only fallow deer (_damhert_) is reported through this flow so far. Legally speaking, this is a game species, and so is included in https://faunabeheer.inbo.be/ . I would not be in favour on including management of this species in the _exotenportaal_. (i) People might get confused on the species' actual status; (ii) best to avoid multiple portals for a same species.

So the question is : what other species are we expecting to emerge through this flow? Raccoon, muntjac and sika deer have dedicated reporting forms with ANB (though for sika, not in the e-loket). Might these come in through WIZ? Any other alien species that we can anticipate for?

**Comment by SanderDevisscher** [comment]
> @SanderDevisscher : I see that only fallow deer (_damhert_) is reported through this flow so far. Legally speaking, this is a game species, and so is included in https://faunabeheer.inbo.be/ . I would not be in favour on including management of this species in the _exotenportaal_. (i) People might get confused on the species' actual status; (ii) best to avoid multiple portals for a same species.



> So the question is : what other species are we expecting to emerge through this flow? Raccoon, muntjac and sika deer have dedicated reporting forms with ANB (though for sika, not in the e-loket). Might these come in through WIZ? Any other alien species that we can anticipate for?

Currently only management of fallow deer has been reported.  But in the future management of other species can be expected through the wiz app flow. I'm not certain which species can be reported at the moment, I'll ask for a list.  
In case of fallow deer I'll add a link to the faunabeheerpagina of fallow deer. With a text in line with: "For management of fallow deer in Flanders see faunabeheer.inbo.be." 
If possible to report management of Canadian goose I would expect the same treatement as fallow deer ? correct ?

**Comment by b-dhondt** [comment]
Yes, I think it's key to come to a list of species for which WIZ applies.

You raise a pertinent case (Canadian goose). Other non-native game species are _fazant_, _moeflon_, (_konijn_)... My argument (i) does apply to them. My argument (ii) does not. _Quo vadis_?

Let me re-consider: I think these species can be included on the _exotenportaal_. But for fallow deer, I would do as you suggest, namely refer to the https://faunabeheer.inbo.be/ . The fallow sits nicely here, among its cousin deer species.

**Comment by SanderDevisscher** [comment]
Sometimes I like to see the world burn 😈 What about sika, axis, muntjac, ... also deer species especially sika for which hybridization with the closely related red deer (already on faunabeheer.inbo.be) is a (possible) impact. Where do they belong ?

**Comment by b-dhondt** [comment]
I would put _sika_, _axis_, _muntjac_ (and ony other non-native Cervidae) under the exotenportaal. So only _fallow_ would be the exception. (Or should we go for double [grofwild + exotenportaal] reporting of _fallow_ after all?)

**Comment by SanderDevisscher** [comment]
double reporting might cause issues => I would link to faunabeheer in case of fallow
